.github/workflows/install_and_test.yml

```diff
@@ -33,3 +33,8 @@ jobs:
         run: pytest
       - name: Run CLI
         run: pyspdxtools -i ./tests/spdx/data/formats/SPDXJSONExample-v2.3.spdx.json
+
+      - name: Install optional dependencies
+        run: python -m pip install networkx
+      - name: Run tests for graph generation
+        run: pytest tests/spdx/test_graph_generation.py
```

Comment:
Can't we just install the optional dependencies together with the rest under "Installation"? Then we wouldn't need to call pytest again here.
By the way, is there a way to make sure that these tests actually are not skipped in the workflow?

Reply:
One approach to enforce the execution of the tests would be to set an environment variable like `PYTEST_DISALLOW_SKIP` and set this in the GitHub Action. Or we could check the output of `pytest` if the tests were executed. As all of this seems rather complicated, I think we can postpone this issue and stick to the current GitHub Action.

Reply:
https://github.com/spdx/tools-python/issues/572